README.md

```diff
@@ -84,22 +86,26 @@ make test
 We use Minikube and Tilt to spawn a local environment that it will reload after any changes inside the charts or the integration code.
 
 Make sure you have these tools or install them:
-- [Install minikube](https://minikube.sigs.k8s.io/docs/start/)
-- [Install Tilt](https://docs.tilt.dev/install.html)
-- [Install Helm](https://helm.sh/docs/intro/install/)
+
+* [Install minikube](https://minikube.sigs.k8s.io/docs/start/)
+* [Install Tilt](https://docs.tilt.dev/install.html)
+* [Install Helm](https://helm.sh/docs/intro/install/)
 
 Create a `values-local.yaml` file from the `values-local.yaml.sample` using a valid license key and your cluster name.
 
 Start the local environment:
+
 ```shell
 make local-env-start
 ```
 
 Notice that local images are build and pushed to docker running inside the minikube cluster since we are running `eval $(minikube docker-env)` before launching Tilt.
 
+Note: for clusters with Kubernetes versions < `1.21`, manually set `apiVersion: batch/v1beta1` for the [e2e-resources/templates/cronjob.yml](https://github.com/newrelic/nri-kubernetes/blob/main/charts/internal/e2e-resources/templates/cronjob.yml).
```

Comment:
This is the only real change in this file. All of the other modifications were automated markdown linting.

Reply:
Why do you need this Note if you have a template with an if statement?

Reply:
this Note should be very specific to local-dev-environment and mention this open issue isthe reason it happens. something like tilt uses `helm template` and `helm template` doesn't render capabilities 
https://github.com/helm/helm/issues/3377

Reply:
The `.Capabilities.APIVersions.Has` if statement is used by E2E testing.